### ISSUE TITLE
Bulkrax: Reference controlled values from Allinson Flex

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/utk-hyku#174]
# Summary

The details of this ticket is WIP. For now it may be ok to ingest the data as is, and we'll update the values later. The [Q has been asked](https://assaydepot.slack.com/archives/C0396LSM06P/p1667836137627239) of the client to determine how clean their data should be. 

Bulkrax has configurations for QA (Questioning Authority). 

Should the configuration should dynamically pull from the specification of the allinson flex profile? 

Or should we just add all of the possible autorities to it, once #170 is complete? 

# Acceptance Criteria

- [ ] 
When a user imports a csv, the qa values should be processed. 

# Screenshots or Video

# Testing Instructions

# Notes

ref: [Allinson Flex x Controlled Values Meeting](https://docs.google.com/document/d/1PDZsGDeZUh9b7BrzsyprmRZOCT4GEwNCg5FXBzyRCUU/edit?usp=sharing)

**Comment by jillpe** [comment]
duplicate of #388
